src/main/java/org/hibernate/infra/replicate/jira/service/jira/JiraService.java

```diff
@@ -307,12 +312,40 @@ public void acknowledge(String project, JiraWebHookEvent event) {
 				throw new ConstraintViolationException("Project " + project + " is not configured.", Set.of());
 			}
 
+			if (context.isUserIgnored(triggeredByUser)) {
```

Comment:
Note this means we will buffer a request body and deserialize the event before we check whether we ignore the request entirely (because of the user).
A potential improvement for later, to reduce the stress on the app, would be to implement this check as a [filter](https://quarkus.io/guides/rest#request-or-response-filters) instead. This might avoid the deserialization -- though probably not the buffering of the request body.
I say for later, because I'm not sure we even need such an optimization without our current workload :)